web/src/containers/admin/LoanRequests.js

```diff
@@ -199,15 +271,15 @@ const LoanRequests = (props) => {
       width: 210,
     },
     {
-      field: "amount",
+      field: "requested_value_atto_dai",
       headerName: "Amount",
       type: "number",
       width: 110,
       // eslint-disable-next-line react/display-name
       renderCell: (props) => {
         return (
           <>
-            <Typography>{props.value}</Typography>
+            <Typography>{props.value / 10 ** 18}</Typography>
```

Comment:
add box with padding

Reply:
Why? I've only changed the value of that field, for the correct value be presented.